### ISSUE TITLE
Incorrect type on errorForConstraint function argument?

### ISSUE THREAD

**Issue by rohanliston** [graphile-contrib/constraint-error-tags#6]
[ConstraintErrorTagsPlugin.d.ts](https://github.com/graphile-contrib/constraint-error-tags/blob/f5febe622b5e221d4332af3c44df56720f2a3fc3/src/ConstraintErrorTagsPlugin.d.ts) defines the `errorForConstraint` function as follows:

```ts
export function errorForConstraint(
  table: string,
  constraint: null
): string | null;
```

Should `constraint` not be `string` here? I'm trying to call the function using a constraint name but TypeScript tells me I should be passing in `null`, which doesn't make sense given what the function is supposed to do.

Happy to submit a PR if I'm on the right track.

**Comment by enisdenjo** [comment]
You're absolutely right! 😅

A PR is very welcome.

**Comment by enisdenjo** [comment]
Released in [v1.0.4](https://github.com/graphile-contrib/constraint-error-tags/releases/tag/v1.0.4).
